### DISCUSSION TITLE
Error when building in Xcode

### DISCUSSION THREAD

**Discussion by gregmaletic** [finnvoor/PlaydateKit#110] [Q&A]
I can build my Playdate game from the command line just fine. But when I try to do it in Xcode, I get what I assume is something that's probably easy to fix... `'pd_api.h' file not found`.

Is it not locating my Playdate SDK?

(Apologies: I know how to write basic apps in Swift with Xcode, but configuring Xcode to do anything else is definitely advanced for me...)

**Comment by finnvoor** [comment]
I'm not quite sure what's going on but I was able to reproduce this occasionally. It seems like the env variables aren't being set in time when building in Xcode. I updated both the Package.swift files in PlaydateKit and PlaydateKitTemplate to try to fix this, You can try to pull both those changes. I'm also guessing it would eventually build for you if you keep clearing derived data / closing and reopening Xcode until it fixes itself, but hopefully these changes make it more consistent.
